class/defaults.yml

```diff
@@ -8,13 +8,13 @@ parameters:
     charts:
       topolvm:
         source: https://topolvm.github.io/topolvm
-        version: v5.0.0
+        version: v6.0.0
 
     images:
       topolvm:
-        registry: quay.io
+        registry: ghcr.io
         repository: topolvm/topolvm-with-sidecar
-        tag: 0.11.0
+        tag: 0.11.1
 
     deviceclasses: []
```

Comment:
https://github.com/topolvm/topolvm/blob/main/charts/topolvm/CHANGELOG.md#600---2022-05-10 is mentioning `This PR changed the type of controller.affinity` Might this have any effect to the component?

Potentially adding the golden tests could help to see, how the Helm chart is rendered.

Reply:
golden tests don't work because of monitoring library dependency... but yes you have to delete the deployments/ds in order to install it.
with the above change, that warants a breaking change and a howto.